op-node/node/node.go

```diff
@@ -426,6 +438,32 @@ func (n *OpNode) initRPCServer(cfg *Config) error {
 	return nil
 }
 
+func (n *OpNode) initHTTPEventStreamServer(cfg *Config) error {
+	eventStream := sse.New()
+	eventStream.AutoReplay = false
+	eventStream.CreateStream("payload_attributes")
+
+	mux := http.NewServeMux()
+	mux.HandleFunc("/events", eventStream.HTTPHandler)
+	addr := net.JoinHostPort(cfg.RPC.ListenAddr, strconv.Itoa(cfg.RPC.ListenPort+1))
```

Comment:
can the rpc server not be reused for the event stream? Otherwise the address and port of the SSE stream should be a flag / config

Reply:
69f4388ec6c33757eea100cb374b035af3de0acb